src/qibojit/backends/gpu.py

```diff
@@ -481,10 +490,20 @@ def __init__(self):
         super().__init__()
         import cuquantum  # pylint: disable=import-error
         from cuquantum import custatevec as cusv  # pylint: disable=import-error
+        from qibo import __version__ as qibo_version
+
+        from qibojit import __version__
 
         self.cuquantum = cuquantum
         self.cusv = cusv
         self.platform = "cuquantum"
+        self.versions = {
+            "qibo": qibo_version,
+            "qibojit": __version__,
+            "numpy": np.__version__,
+            "cupy": self.cp.__version__,
+            "cuquantum": self.cuquantum.__version__,
+        }
```

Comment:
Same as above
```suggestion
        self.versions["cuquantum"] = self.cuquantum.__version__
```
you can use all the definitions from CupyBackend and only add the cuquantum version here.